runtime/src/main/java/io/quarkiverse/renarde/htmx/HxController.java

```diff
@@ -0,0 +1,118 @@
+package io.quarkiverse.renarde.htmx;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.WebApplicationException;
+import jakarta.ws.rs.core.HttpHeaders;
+import jakarta.ws.rs.core.Response;
+
+import io.quarkiverse.renarde.Controller;
+import io.quarkus.qute.Qute;
+import io.quarkus.qute.TemplateInstance;
+import io.vertx.core.http.HttpServerResponse;
+
+/**
+ * This is a Controller's overlay to simplify usage of Quarkus Renarde alongside {@see <a href="https://htmx.org/">htmx</a>}.
+ */
+public class HxController extends Controller {
```

Comment:
Shouldn't this be `abstract`?

Reply:
@FroMage If I remember well I reproduced Controller which is not abstract right? But yes I could switch to abstract..

Reply:
Well, OK, I see. But, `Controller` never needed to be abstract because it's not a controller itself :) It's not even annotated with `@Path` so, there's nothing special.

Your `HxController` is a subtype of `Controller` so it gets picked up to be a controller and a `@Path` will be added to it by our processor.

Which is why you're excluding it.

Now that I looked at the processor, we still scan abstract controllers in case they define public endpoints. Which is not the case for `HxController`. So, probably I'd say it's better to be safe and make both `Controller` and `HxController` `abstract` and keep this one as `excluded` as you did, since you're not defining endpoints, and are also not using any redirects.